lib/health_cards/health_card.rb

```diff
@@ -98,8 +143,22 @@ def types
       def supports_type?(*type)
         !types.intersection(type).empty?
       end
+
+      protected
+
+      def parent_allowables(base = {})
+        self < HealthCards::HealthCard ? superclass.parent_allowables(base.merge(superclass.allowable)) : base
+      end
+
+      def parent_disallowables(base = {})
+        self < HealthCards::HealthCard ? superclass.parent_disallowables(base.merge(superclass.disallowable)) : base
```

Comment:
`superclass.disallowable` should already contain `superclass.parent_disallowables`

```suggestion
        self < HealthCards::HealthCard ? base.merge(superclass.disallowable) : base
```